addons/web/static/src/legacy/js/libs/jquery.js

```diff
@@ -147,6 +147,33 @@ $.fn.extend({
         }
         return $el;
     },
+    /**
+     * Adapt the given css property by adding the size of a scrollbar if any.
+     * Limitation: only works if the given css property is not already used as
+     * inline style for another reason.
+     *
+     * @param {boolean} [add=true]
+     * @param {boolean} [isScrollElement=true]
+     * @param {string} [cssProperty='padding-right']
+     */
+    compensateScrollbar(add = true, isScrollElement = true, cssProperty = 'padding-right') {
```

Comment:
Actually it should be `padding-left` if LTR.. I will change it

Reply:
Note that this is a problem that also exists in V14 and V15. It will also be necessary to adapt `compensateScrollbar` in these versions.

Reply:
> Note that this is a problem that also exists in V14 and V15. It will also be necessary to adapt `compensateScrollbar` in these versions.

This is done in this PR https://github.com/odoo/odoo/pull/126506